specification/ai/Azure.AI.Unified/models/models.tsp

```diff
@@ -0,0 +1,73 @@
+import "@typespec/rest";
+import "@azure-tools/typespec-autorest";
+import "@typespec/versioning";
+import "@azure-tools/typespec-azure-core";
+import "../main.tsp";
+import "../common/models.tsp";
+import "@typespec/openapi";
+
+using TypeSpec.OpenAPI;
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using TypeSpec.Versioning;
+using Azure.Core;
+using Azure.Core.Traits;
+
+namespace Azure.AI.Unified;
+
+#suppress "@azure-tools/typespec-azure-core/composition-over-inheritance"
+@doc("Model Deployment Definition")
+model Model {
```

Comment:
Nothing is optional in Model? Everything below must be populated?

Reply:
All of this information is already in the control plane object that will be used to return this information (except connection name, which we will have to add manually). Do you think it's still worth marking things as optional in that case?